test/integration/http/snapshots/github-webhook-payload-test.js.md

```diff
@@ -1,26 +1,25 @@
-# Snapshot report for `test/integration/github-webhook-payload-test.js`
+# Snapshot report for `test/integration/http/github-webhook-payload-test.js`
```

Comment:
These tests look different, but ultimately the snapshot generation just got rearranged because I moved the file to a different location and the tests run in parallel.